src/main/com/fulcrologic/guardrails/core.cljc

```diff
@@ -664,7 +667,7 @@
        {:arglists '([name doc-string? attr-map? [params*] gspec prepost-map? body?]
                     [name doc-string? attr-map? ([params*] gspec prepost-map? body?) + attr-map?])}
        [& forms]
-       (if (cfg/get-env-config)
+       (if (emit-specs?)
```

Comment:
This is a behavioral change that I don't think is correct. You may still want an *instrumented* function, without emitting specs.  Guardrails does NOT use st/instrument like ghostwheel, it rewrites the funcions.

Reply:
Good catch, I just blindly generalized/replaced this as the shape of the code looked similar, fixing.